cmd/cosign/cli/fulcio/fulcioroots/fulcioroots.go

```diff
@@ -64,11 +64,16 @@ func initRoots() *x509.CertPool {
 		// Retrieve from the embedded or cached TUF root. If expired, a network
 		// call is made to update the root.
 		ctx := context.Background() // TODO: pass in context?
-		for _, fulcioTarget := range []string{fulcioTargetStr, fulcioV1TargetStr} {
-			buf := tuf.ByteDestination{Buffer: &bytes.Buffer{}}
-			if err := tuf.GetTarget(ctx, fulcioTarget, &buf); err != nil {
-				panic(errors.Wrap(err, "creating root cert pool"))
-			}
+		buf := tuf.ByteDestination{Buffer: &bytes.Buffer{}}
```

Comment:
WDYT about keeping the previous code and use a bool to panic if one of the targets wasn't added?
(as in gracefully fail, just fail when none of the errors were nil)

Reply:
make sense, I will update the patch